### DISCUSSION TITLE
[BUG] Docker version can't URL one-click login

### DISCUSSION THREAD

**Discussion by kidzgy** [transmission/transmission#6771] [Q&A]
### What is the issue?

I installed docker version linuxserver/transmission on April 2, 2024 and all configurations have been debugged to the point where I can download and upload normally.

What the title means by one-click login is: http://admin:admin@192.168.1.100:9091 A link like this skips the account password entry window and goes directly to the Transmission interface.

If you are using the Transmission package version, the one-click login will work. However, if you are using the docker version of Transmission, the one-click login will not work, and you will need to manually enter your account password using http://192.168.1.100:9091 instead.

The failed screen is as follows:
![20240403103413](https://github.com/transmission/transmission/assets/6967810/93492150-5d7f-45e5-80dc-d1e67a23acf2)

In the failure screen, even if you press F5 or CTRL+F5 to refresh, it still fails, but if you open http://192.168.1.100:9091 in a new tab, you don't need to enter your account password anymore. 

The success screen of manually entering the password is as follows:
![20240403103444](https://github.com/transmission/transmission/assets/6967810/b6896789-e12b-4c22-863c-ecd15aab0719)

### Which application of Transmission?

docker linuxserver/transmission

### Which version of Transmission?

4.0.5

**Comment by tearfur** [comment]
Since this does not seem like Tranmission's fault, I will move this to discussions.
